### PR TITLE
Make the grace period configurable

### PR DESCRIPTION
**Pull request by Taction**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Add content for making the grace period configurable.

## Issue reference

Please reference the issue this PR will close: #2024

### CODE REVIEW

File: daprdocs/content/en/reference/arguments-annotations-overview.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ This table is meant to help users understand the equivalent options for running
 | `--app-protocol` | `--app-protocol` | `-P` | `dapr.io/app-protocol` | Tells Dapr which protocol your application is using. Valid options are `http` and `grpc`. Default is `http` |
 | `--sentry-address` | `--sentry-address` | | not supported | Address for the Sentry CA service |
 | `--version` | `--version` | `-v` | not supported | Prints the runtime version |
+| `--dapr-graceful-shutdown-seconds` | not supported | | `dapr.io/graceful-shutdown-seconds` | Graceful shutdown duration in seconds for dapr, the maximum duration before force shutdown when waitting all in-progress requests to complete. Defaults to `5`, and maximum to `30` which is equivalent to Kubernetes termination grace period. Any vaule larger than the minimum duration will be setting to the minimum duration.  |
```
Comment (willtsai) [inline daprdocs/content/en/reference/arguments-annotations-overview.md]:
```suggestion
| `--dapr-graceful-shutdown-seconds` | not supported | | `dapr.io/graceful-shutdown-seconds` | Graceful shutdown duration in seconds for Dapr, the maximum duration before forced shutdown when waiting for all in-progress requests to complete. Defaults to `5`, and maximum value is `30` which is equivalent to Kubernetes termination grace period. Any value larger than the minimum duration will be set to the minimum duration.  |
```

Comment (willtsai) [inline daprdocs/content/en/reference/arguments-annotations-overview.md]: Additionally, please re-evaluate this sentence as I think you might either mean "any value larger than the maximum" or "any value smaller than the minimum"?
`Any vaule larger than the minimum duration will be setting to the minimum duration.`

Comment (Taction) [reply daprdocs/content/en/reference/arguments-annotations-overview.md]: done

Comment (Taction) [reply daprdocs/content/en/reference/arguments-annotations-overview.md]: Yes, thank you for pointing it out. What I really want to describe is`any value larger than the maximum`.
done.

Comment (msfussell) [reply daprdocs/content/en/reference/arguments-annotations-overview.md]: @Taction  - Can you accept the suggested changes from @willtsai

Comment (Taction) [reply daprdocs/content/en/reference/arguments-annotations-overview.md]: I have accepted these suggestions in fact, but the code has a minor change, so this doc has been changed too. 
I'll fix dco asap.

Comment (Taction) [reply daprdocs/content/en/reference/arguments-annotations-overview.md]: The limit of `maximum duration` has been removed after a discussion about it.

**Review by greenie-msft — APPROVED** [review]

**Comment by greenie-msft** [comment]
LGTM.

**Pull request merged**